src/cljs/rems/application.cljs

```diff
@@ -617,31 +618,43 @@
   `:element-id`         - id of the element to generate unique ids
   `:attributes`         - user attributes to display
   `:application`        - application
-  `:group?`             - specifies if a group border is rendered
-  `:can-remove?`        - can the user be removed?
-  `:accepted-licenses?` - has the member accepted the licenses?"
-  [{:keys [element-id attributes application group? can-remove? accepted-licenses? invited-user?]}]
+  `:group?`             - specifies if a group border is rendered"
+  [{:keys [element-id attributes application group?]}]
```

Comment:
Because member-info gets the whole application, I was able to just push all the logic inside it. It might be better to split this into a view component and a container that does the logic, but then we'd be backin the world of `can-remove?` parameters.

Reply:
I think there are some higher order components that would make sense in the future, but no need to complicate this PR. I'll add an idea task.

Reply:
https://github.com/CSCfi/rems/issues/2659